org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/util/ObjectUtility.java

```diff
@@ -205,4 +207,30 @@ public static boolean isOneOf(Object o, Collection<?> elements) {
     }
     return false;
   }
+
+  /**
+   * @return {@code true} if the given object represents an actual value as opposed to an empty value such as null,
+   *         empty optional, empty collection, empty string etc.
+   */
+  public static boolean hasValue(Object o) {
```

Comment:
I don't know if that is too great of an idea either. People are going to rely on it and probably don't expect a true here "just because we don't know". I'd much rather prefer we do not try to find out if an xml entry has a value or not.